freeqdsk/geqdsk.py

```diff
@@ -182,6 +183,163 @@
 )
 
 
+def _synonym(canonical: str) -> property:
+    """Set a property which is just a synonym for another attribute"""
+
+    return property(
+        lambda self: getattr(self, canonical),
+        lambda self, value: setattr(self, canonical, value),
```

Comment:
Could also set `doc=f"Synonym for {canonical}"`

Reply:
Nice, great idea!